paradox-nakadi-consumer-core/src/test/java/de/zalando/paradox/nakadi/consumer/core/client/impl/ClientImplTest.java

```diff
@@ -19,25 +18,49 @@
 import de.zalando.paradox.nakadi.consumer.core.domain.NakadiPartition;
 import de.zalando.paradox.nakadi.consumer.core.http.handlers.testdomain.OrderReceived;
 
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+
 import rx.Single;
 
 public class ClientImplTest {
 
-    private static final String NAKADI_URL = "http://localhost:8080";
+    private static final int NAKADI_PORT = 8081;
```

Comment:
Starting on a specific port is not needed

```
  /**
   * Starts the server on the loopback interface for the given port.
   *
   * @param port the port to listen to, or 0 for any available port. Automated tests should always
   * use port 0 to avoid flakiness when a specific port is unavailable.
   */
```

To obtain the url, use  `mockNakadi.url("/").toString()`